configuration/restdocs.md

```diff
@@ -44,8 +44,18 @@ curl -X GET --header "Accept: application/json" "http://192.168.1.35:8080/rest/s
 ```
 
 The commands above have been copied from the REST API documentation for illustration. 
+
+## REST API documentation
+
+The REST API documentation is available on your openHAB installation as a custom user interface.
```

Comment:
This interface combines documentation and the possibility to interact and experiment with the API.